test/14.js

```diff
@@ -0,0 +1,13 @@
+const expect = require('chai').expect;
+const diagnal = require('../solutions/14');
+const solution = diagnal.solution;
+// const yourSolution = "";
+
```

Comment:
let solution = require('../solutions/14').solution;
//solution = require('../yourSolution').solution;

and remove the //const yourSolution line
try this method!